ci/run_clang_tidy.sh

```diff
@@ -45,6 +45,11 @@ function exclude_testdata() {
   grep -v tools/testdata/check_format/
 }
 
+# Do not run clang-tidy on envoy_headersplit testdata files.
+function exclude_testdata() {
```

Comment:
I think that you redefined the exclude_testdata function and effectively undid the tools/testdata/check_format exclusion a few lines up.

Reply:
Nice catch! Thanks for fix it